price-fetcher.py

```diff
@@ -0,0 +1,41 @@
+import requests
```

Comment:
the simulation environment itself does not need real time data, at least at this stage

we need to be able to quickly simulate various scenarios -- it's better to just generate a price series with some stochastic process 

but can leave it for now, not a core component for the amm environment